Resources/Prototypes/Entities/Mobs/NPCs/space.yml

```diff
@@ -301,3 +301,109 @@
     name: ghost-role-information-salvage-spider-name
     description: ghost-role-information-salvage-spider-description
   - type: SalvageMobRestrictions
+
+- type: entity
+  name: space cobra
+  id: MobCobraSpace
+  parent: MobSpaceBasic
+  description: Long fangs and a glowing hood, and the alluring look begs to come closer.
+  components:
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/Animals/spacecobra.rsi
+    layers:
+    - map: [ "enum.DamageStateVisualLayers.Base" ]
+      state: space_cobra
+    - map: [ "enum.DamageStateVisualLayers.BaseUnshaded" ]
+      state: glow
+      shader: unshaded
+  - type: FootstepModifier
+    footstepSoundCollection:
+      collection: FootstepSnake
+  - type: MobThresholds
+    thresholds:
+      0: Alive
+      75: Critical
+      100: Dead
+  - type: Stamina
+    excess: 150
+  - type: DamageStateVisuals
+    states:
+      Alive:
+        Base: space_cobra
+        BaseUnshaded: glow
+      Critical:
+        Base: dead_space_cobra
+      Dead:
+        Base: dead_space_cobra
```

Comment:
Mobs shouldn't have crit states.

Reply:
Yeah, the other ones need removing too at some point.

Reply:
Removed the crits from space mobs